commandreporter/command_reporter.go

```diff
@@ -36,10 +35,6 @@ func NewCommandReporter(writers ...io.Writer) *CommandReporter {
 func (r *CommandReporter) Report(startTime time.Time, cmd *exec.Cmd) {
 	startColor := ""
 	endColor := ""
-	if !config.DefaultReporterConfig.NoColor {
```

Comment:
The `github.com/onsi/ginkgo/config` package [has been deprecated](https://github.com/onsi/ginkgo/blob/master/config/deprecated.go).

We're directed to start using the `GinkgoConfiguration()` function to get copies of the current configuration. So this if statement could theoretically be changed to something like the following:
```golang
_, reporterConfig := GinkgoConfiguration()
if !reporterConfig.NoColor {
...
}
```

Reply:
I'll take a look sometime next week, thanks for the feedback!  If anyone else sees this and is a maintainer and wants to do this faster than I'll take a look, I checked the box for allow edits by maintainers so feel free to make your own pushes as well